sherpa/plot/__init__.py

```diff
@@ -564,13 +585,35 @@ def plottop(self, plot, *args, **kwargs):
         clearaxes = kwargs.get('clearwindow', True)
         self._clear_window(0, 0, clearaxes)
 
+        """
         # FIXME: should not know about FitPlot, terrible hack to remove label
```

Comment:
Do we need/want to leave the old code and FIXME note or should we delete it?

Reply:
I think we can probably remove this comment: the following `isinstance` check is a sign that something is less-than ideal.